meta-swupd/classes/swupdbundle.bbclass

```diff
@@ -44,7 +44,8 @@ python swupdbundle_virtclass_handler () {
         basebundle = bundle
 
     # Not producing any real images, only the rootfs directory.
-    e.data.setVar("IMAGE_FSTYPES", "")
+    e.data.setVarFlag("do_image", "noexec", "1")
```

Comment:
"swupdbundle: disable image construction differently" breaks Ostro OS because do_image does critical parts of the image building (IMA signing, xattr_images_fix_transmute()).

@incandescant please revert in meta-swupd and try something else.